hierarchy/store.go

```diff
@@ -59,6 +59,21 @@ func (store *Store) BuildHierarchy(instanceID, codeListID, dimensionName string)
 		return err
 	}
 
+	markNodesToRemain(connection, instanceID, dimensionName)
+	if err != nil {
```

Comment:
the errors returned from these functions aren't being captured, so currently this check is not valuable

Reply:
Good spot! I have updated to capture errors 👍